### ISSUE TITLE
URLs in datasets description preview not rendered

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#333]
**URLs in datasets description preview not rendered**

URLs starting with `http://` are nicely rendered as links in any of the metadata of a dataset. This is however not the case in the description preview on the top of a dataset page: you&#39;ll have to click `more` to see them as links. I don&#39;t know if this is intentional, but it looks a bit odd.

-----
fbitem-dataset83e20573-f7dd-4852-9159-21566e1e691e
Reported by: @peterdesmet
System: Chrome 59.0.3071 / Mac OS X 10.12.5
Referer: https://www.gbif.org/dataset/83e20573-f7dd-4852-9159-21566e1e691e
Window size: width 1229 - height 736
[API log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-08-17T12:29:38.862Z',mode:absolute,to:'2017-08-17T12:35:38.862Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499%20AND%20(request:%22%2F%2Fapi.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)
[Site log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-08-17T12:29:38.862Z',mode:absolute,to:'2017-08-17T12:35:38.862Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E399%20AND%20(request:%22%2F%2Fdemo.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)

**Comment by peterdesmet** [comment]
Screenshot for context:

<img width="787" alt="screen shot 2017-08-17 at 14 29 58" src="https://user-images.githubusercontent.com/600993/29412484-4c3d1e2e-8359-11e7-8b09-cf53caaddced.png">

**Comment by MortenHofft** [comment]
Thanks
It is intentionaly excluded because the library for truncating html didn't handle your description very well. It worked for all other description i tried, but for some reason not yours. So until a stable solution is found this is intentional excluded. 
But yes, it would be nice.

**Comment by gbif-portal** [comment]
Closed and transferred to  https://github.com/gbif/portal16/issues/533

**Comment by peterdesmet** [comment]
@MortenHofft the odd thing is that it is handled correctly (for the exact same text) on the full description lower on the page: https://www.gbif.org/dataset/83e20573-f7dd-4852-9159-21566e1e691e#description

**Comment by MortenHofft** [comment]
I disabled it for the shortened description because the used library failed. The failing part was truncating the html, not linkifying the text. Only the shortened description on the top is truncated. The html truncater worked well in general, but failed sometimes. I believe your dataset was actually the one it couldn't handle. So I disabled it across the site until I could find a more stable solution.
